### ISSUE TITLE
Unable to satisfy ExcludeAssets=runtime error message with MSBuildLocator

### ISSUE THREAD

**Issue by ericnewton76** [microsoft/MSBuildLocator#56]
Originally, I was instructed to add PackageReferences to Microsoft.Build Microsoft.Build.Framework Microsoft.Build.Tasks.Core when using MSBuildWorkspace.Create() from Roslyn's `Microsoft.CodeAnalysis.Workspaces.MSBuild` package.

This MSBuildLocator makes more sense, because I dont wanna drag around MSBuild DLLs with me... should use current system's msbuild.

However, now I'm getting the error message about needing to specify ExcludeAssets=runtime on my PackageReference for Microsoft.Build.*** packages.   Added via commit e81f3666

So i attempted to fix it both of these ways, which both still failed the validation check.

This way didn't work:

```xml
    <PackageReference Include="Microsoft.Build" ExcludeAssets="runtime">
      <Version>15.8.166</Version>
    </PackageReference>
```

and this way didn't work, either:

```xml
    <PackageReference Include="Microsoft.Build">
      <Version>15.8.166</Version>
      <ExcludeAssets>runtime</ExcludeAssets>
    </PackageReference>
```

My csproj is Net Desktop Console app, so my PackageReferences look like this:

```xml
    <PackageReference Include="Microsoft.Build">
      <Version>15.8.166</Version>
    </PackageReference>
    <PackageReference Include="Microsoft.Build.Framework">
      <Version>15.8.166</Version>
    </PackageReference>
    <PackageReference Include="Microsoft.Build.Tasks.Core">
      <Version>15.8.166</Version>
    <PackageReference Include="Microsoft.Build.Locator">
      <Version>1.1.2</Version>
    </PackageReference>

```

**Comment by ericnewton76** [comment]
Btw, for my project, I actually dont want any of the MSBuild references (besides the Locator) so I removed them all from my project, but I'm posting here because another might need them referenced.

**Comment by rainersigwald** [comment]
I tried just now with a new template Windows Console Application, adding your exact PackageReferences, which showed the error. Then I added `ExcludeAssets="runtime"` to each reference and it went away, as I expected/hoped. So I'm curious what was different for you.

That's surprising! Both of those forms should have worked to specify that metadata. Where were you seeing the error--in VS, or on command-line builds? Did you try explicitly restoring (or `git clean` + build again)?
